packages/utils/src/attestations.test.ts

```diff
@@ -71,17 +63,9 @@ describe('Attestation Utils', () => {
       'Celo attestation code: T7p-Mn1_L5zJuycAgAxYVqaJDp5r2TORcb775fdVoARbdJ-rNP-LArWCNJzJ6KjuVg0yskkEM8vVtl1PPmOsWwE='
 
     it('sanitizes correctly', () => {
-      expect(sanitizeBase64(CODE_1)).toBe(SANITIZED_CODE_1)
-      expect(sanitizeBase64(CODE_2)).toBe(SANITIZED_CODE_2)
-      expect(sanitizeBase64(NORMAL_CODE)).toBe(NORMAL_CODE)
-    })
-  })
-
-  describe('decodeAttestationCode', () => {
-    it('decodes the base64 message correctly', () => {
-      expect(decodeAttestationCode(MESSAGE_1_CODE)).toBe(MESSAGE_1_DECODED)
-      expect(decodeAttestationCode(MESSAGE_2_CODE)).toBe(MESSAGE_2_DECODED)
-      expect(decodeAttestationCode(MESSAGE_3_CODE)).toBe(MESSAGE_3_DECODED)
+      expect(sanitizeMessageBase64(CODE_1)).toBe(SANITIZED_CODE_1)
```

Comment:
It would be great to update these test cases with the deeplink prefixes

Reply:
left a comment to this effect